docs/containerd-env-setup.md

```diff
@@ -4,23 +4,23 @@ This document will walk through how to setup a nydus image service to work with
 
 ## Install All Nydus Binaries
 
-Get `nydus-image`, `nydusd`, `nydusify`, `ctr-remote` and `nydus-overlayfs` binaries from [release](https://github.com/dragonflyoss/image-service/releases/latest) page.
+1. Get `nydus-image`, `nydusd`, `nydusify`, `ctr-remote` and `nydus-overlayfs` binaries from [release](https://github.com/dragonflyoss/image-service/releases/latest) page.
 
 ```bash
-sudo cp nydusd nydus-image /usr/local/bin
-sudo cp nydusify containerd-nydus-grpc /usr/local/bin
-sudo cp ctr-remote nydus-overlayfs /usr/local/bin
+sudo cp nydusd nydus-image nydusify ctr-remote nydus-overlayfs /usr/local/bin
 ```
 
-Get `containerd-nydus-grpc` binary from nydus-snapshotter [release](https://github.com/containerd/nydus-snapshotter/releases) page.
+2. Get `containerd-nydus-grpc` (nydus snapshotter) binary from nydus-snapshotter [release](https://github.com/containerd/nydus-snapshotter/releases) page.
 
 ```bash
-sudo cp nydusify containerd-nydus-grpc /usr/local/bin
+sudo cp containerd-nydus-grpc /usr/local/bin
```

Comment:
ditto

Reply:
It will be very convinient if we paste a nyuds-snapshotter latest release download link here.

Reply:
1. Fixed.
2. Maybe the latest release is not stable?

Reply:
I can mark RC release as `pre-release`, which means it is stable